app/models/event_instance.rb

```diff
@@ -38,12 +39,25 @@ def start_datetime
 
   def tweet_hangout_notification
     case self.category
-    when 'Scrum'
-      TwitterService.tweet("#Scrum meeting with our #distributedteam is live on #{hangout_url} Join in and learn about our #opensource #projects!")
-    when 'PairProgramming'
-      TwitterService.tweet("Pair programming on #{self.project.title} #{hangout_url} #pairwithme, #distributedteam")
-    else
-      Rails.logger.error "''#{self.category}'' is not a recognized event category for Twitter notifications. Must be one of: 'Scrum', 'PairProgramming'"
+      when 'Scrum'
+        TwitterService.tweet("#Scrum meeting with our #distributedteam is live on #{hangout_url} Join in and learn about our #opensource #projects!")
+      when 'PairProgramming'
+        TwitterService.tweet("Pair programming on #{self.project.title} #{hangout_url} #pairwithme, #distributedteam")
+      else
+        Rails.logger.error "''#{self.category}'' is not a recognized event category for Twitter notifications. Must be one of: 'Scrum', 'PairProgramming'"
     end
   end
+
+  def tweet_yt_link
+    case self.category
+      when 'Scrum'
+        TwitterService.tweet("#{broadcaster} just hosted an online #scrum using ##googlehangouts Missed it? Catch the recording at youtu.be/#{self.yt_video_id} #opensource")
```

Comment:
i'm wondering is the twitter message templates should move to a yaml file or are by some other means separated from the model? phase 3 maybe?